test/markup/typescript/namespace_as_identifier.expect.txt

```diff
@@ -0,0 +1,4 @@
+
+            <span class="hljs-keyword">const</span> message = <span class="hljs-string">&#x27;foo&#x27;</span>;
+            <span class="hljs-keyword">const</span> <span class="hljs-keyword">namespace</span> = <span class="hljs-string">&#x27;bar&#x27;</span>;
```

Comment:
This isn't the desire behavior... We'll also need to remove it from `TS_SPECIFIC_KEYWORDS` (lets just comment it out - and mention it has a separate rule below).